addons/l10n_eu_oss/models/res_company.py

```diff
@@ -111,28 +111,43 @@ def _get_repartition_lines_oss(self):
     def _get_oss_account(self):
         self.ensure_one()
         if not self.env.ref(f'l10n_eu_oss.oss_tax_account_company_{self.id}', raise_if_not_found=False):
+            self._create_oss_account()
+        return self.env.ref(f'l10n_eu_oss.oss_tax_account_company_{self.id}')
+
+    def _create_oss_account(self):
+        known_oss_accounts = {
+            'de_skr03': 'account_1767',
+            'de_skr04': 'chart_skr04_3817',
+        }
+        if (self.chart_template in known_oss_accounts
+            and (oss_account_if_exists :=
+                self.env['account.chart.template'].with_company(self).ref(known_oss_accounts[self.chart_template],
+                                                                          raise_if_not_found=False))):
```

Comment:
:pray: 
```suggestion
        if (
            self.chart_template in known_oss_accounts
            and (oss_account_if_exists :=
                self.env['account.chart.template'].with_company(self).ref(
                    known_oss_accounts[self.chart_template],
                    raise_if_not_found=False,
                )
            )
        ):
```

Or at least

```suggestion
        if (self.chart_template in known_oss_accounts and (oss_account_if_exists :=
            self.env['account.chart.template'].with_company(self).ref(
                known_oss_accounts[self.chart_template],
                raise_if_not_found=False,
            )
        )):
```